src/main/java/duke/Tasklist.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * The Tasklist class encapsulates
+ * the task list and all operations on it
+ */
+
+public class Tasklist {
+	ArrayList<Task> listOfThings = new ArrayList<Task>();
```

Comment:
Great! You distinguish clearly between single-valued and multi-valued variables.

Reply:
Great job on variable naming between single valued and multi valued variables